src/Framework.Configurator/Handlers/UpdateBusinessRoleHandler.cs

```diff
@@ -15,53 +15,54 @@
 
 using Microsoft.AspNetCore.Http;
 
-namespace Framework.Configurator.Handlers
+namespace Framework.Configurator.Handlers;
+
+public class UpdateBusinessRoleHandler : BaseWriteHandler, IUpdateBusinessRoleHandler
 {
-    public class UpdateBusinessRoleHandler: BaseWriteHandler, IUpdateBusinessRoleHandler
-    {
-        private readonly IAuthorizationBLLContext authorizationBllContext;
+    private readonly IAuthorizationBLLContext authorizationBllContext;
 
-        public UpdateBusinessRoleHandler(IAuthorizationBLLContext authorizationBllContext) => this.authorizationBllContext = authorizationBllContext;
+    public UpdateBusinessRoleHandler(IAuthorizationBLLContext authorizationBllContext) =>
+            this.authorizationBllContext = authorizationBllContext;
 
-        public async Task Execute(HttpContext context)
-        {
-            var roleId = (string)context.Request.RouteValues["id"];
-            var role = await this.ParseRequestBodyAsync<RequestBodyDto>(context);
+    public async Task Execute(HttpContext context)
+    {
+        var roleId = (string)context.Request.RouteValues["id"] ?? throw new InvalidOperationException();
+        var role = await this.ParseRequestBodyAsync<RequestBodyDto>(context);
 
-            this.Update(new Guid(roleId), role);
-        }
+        this.Update(new Guid(roleId), role);
+    }
 
-        [SuppressMessage("SonarQube", "S2436", Justification = "It's ok. BusinessRoleOperationLink automatically link to BusinessRole")]
-        private void Update(Guid id, RequestBodyDto role)
-        {
-            var businessRoleBll = this.authorizationBllContext.Authorization.Logics
-                                      .BusinessRoleFactory.Create(BLLSecurityMode.Edit);
-            
-            var domainObject = businessRoleBll.GetById(id, true);
-            var mergeResult =
-                    domainObject.BusinessRoleOperationLinks.GetMergeResult(
-                     role.OperationIds,
-                     s => s.Operation.Id,
-                     s => s);
+    [SuppressMessage("SonarQube", "S2436", Justification = "It's ok. BusinessRoleOperationLink automatically link to BusinessRole")]
```

Comment:
this isn't working for the cloud sonar, and may be redundant after adding `var _ = ***`